internal/operators/ocs/validations.go

```diff
@@ -110,6 +110,20 @@ func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) bool {
 		return false
 	} else {
 		for _, host := range hosts { // if the worker nodes >=3 , install OCS on all the worker nodes if they satisfy OCS requirements
+
+			// if number of hosts are >=6 and one of them has Role as Auto-assign
+			if host.Role == models.HostRoleAutoAssign {
+				status = "Not supporting OCS installation if any 1 host has role Auto-assign for number of host >=6"
```

Comment:
@avishayt user facing message, might want to take a look

Reply:
Thanks @omertuc 

I prefer: "All host roles must be assigned to enable OCS"

Reply:
Done